### ISSUE TITLE
Didn't give my main Discord 'till last issue was closed

### ISSUE THREAD

**Issue by pooberu** [Dean2k/SARS#2]
I don't know if you saw it considering the last issue was closed before I gave it. It's wheezin_
If it was already seen and just not gotten around to yet then I am so sorry!

**Comment by pooberu** [comment]
I'm still not unbanned, which is why I created this.

**Comment by Dean2k** [comment]
![image](https://github.com/Dean2k/SARS/assets/5357629/5a9b3788-dedf-4818-9842-658774b7831a)
Yes, you are i did it yesterday.

**Comment by pooberu** [comment]
Oh. I'm not sure why I'm unable to join the server then.

**Comment by pooberu** [comment]
![image](https://github.com/Dean2k/SARS/assets/144723894/736b94ce-2a32-4e1d-bd46-0e9e9ba172fd)

**Comment by pooberu** [comment]
Yeah, so, I don't know how I'm gonna fix that considering even though my main is unbanned, it still has the phone number associated with my account banned through my alt.

**Comment by pooberu** [comment]
I've come to the conclusion that I may end up just being flat out unable to join the server unless you can unban my alt and I just don't join on it again. My phone number was associated with both accounts and that was a major goof-up on my part.

**Comment by pooberu** [comment]
Would there be any chance of my alt being unbanned as long as I don't rejoin on it? It said on multiple places that if your phone number is used to verify on two accounts at some point, no matter if one is banned or not, if another account has that number associated with it then it'll be banned as well if only one is banned from a server.
